pom.xml

```diff
@@ -277,6 +251,15 @@
                             <rpt>SCRIPT_STYLE</rpt>
                         </mapping>
                     </configuration>
+                    <dependencies>
+                        <dependency>
+                            <!-- includes common copyright headers for sources -->
+                            <groupId>org.kaazing</groupId>
+                            <artifactId>community.license</artifactId>
+                            <type>jar</type>
+                            <version>${project.version}</version>
```

Comment:
This fails.  For example, if you're trying to build nuklei 7.19, then it will end up looking for artifact org.kaazing:community.license:jar:7.19.